timApp/static/stylesheets/style.scss

```
@@ -53,6 +53,11 @@ tim-dialog-frame {
     position: static;
     background-color: var(--bg-color);
 
+    &.draggable-detached {
+        // need calc here because vanilla Sass does not support min(), max() etc. with differing units
+        width: $paragraph-preferred-width;
```

Mielellään käytetään CSS-muuttujaa ellei ole syytä juuri SCSS-muuttujan käyttöön:

```suggestion
        width: var(--paragraph-preferred-width);
```

CSS-muuttujia voi muokata sitten dokumenttikohtaisesti tekemättä uusia teemoja.